trin-core/src/types/block_body.rs

```diff
@@ -200,80 +269,144 @@ pub struct StorageKeys {
 mod tests {
     use super::*;
     use rstest::rstest;
+    use ssz::{Decode, Encode};
 
     // tx data from: https://etherscan.io/txs?block=14764013
     #[rstest]
-    #[case(TX1, 102)]
-    #[case(TX2, 1078)]
     #[case(TX3, 454)]
-    #[case(TX4, 455)]
```

Comment:
Maybe add a comment explaining why this keeps the transactions that it does. Want to make sure there's coverage of typed and un-typed of course.